src/promptflow-tracing/promptflow/tracing/_start_trace.py

```diff
@@ -108,12 +108,17 @@ def start_trace(
             start_trace_with_devkit(collection=collection, attributes=_attributes, run=_run)
 
 
-def setup_exporter_from_environ() -> None:
+def setup_exporter_from_environ(only_inject_openai: bool = False) -> None:
     # openai instrumentation
     logging.debug("injecting OpenAI API...")
     inject_openai_api()
```

Comment:
shall we move the inject_oepn_api out of the setup_exporter_from_environ function? 
So we don't need to add this strance bool parameter here

Reply:
That also makes sense to me,  @thy09  @huaiyan any comments on Clement's feedback?

Reply:
updated based on comments